scapy/arch/windows/compatibility.py

```diff
@@ -248,3 +248,7 @@ def sniff(count=0, store=1, offline=None, prn = None, lfilter=None, L2socket=Non
 
 import scapy.sendrecv
 scapy.sendrecv.sniff = sniff
+
+# If wpcap.dll is not available
+if (not conf.use_winpcapy) and (not conf.use_pcap) and (not conf.use_dnet):
```

Comment:
Can you write the test as `if not (conf.use_winpcapy or conf.use_pcap or conf.use_dnet):`